assets/css/main.scss

```diff
@@ -186,3 +186,105 @@ footer {
         display: none;
     }
 }
+
+.hall-of-fame{
+    padding: 15px;
+    text-align: center;
+
+    img {
+        width: 100%;
+        padding: 5px;
+        border-radius: 25px;
+    }
+    .contact-icon{
+        width: 15px;
+        padding: 0px;
+    }
+
+    .thumbnail{
+        display: block;
+        background-color: white;
+        border: 1px solid #ddd;
+        line-height: 1.6;
+        border-radius: 25px;
+        -webkit-transition: border 0.2s ease-in-out;
+        -o-transition: border 0.2s ease-in-out;
+        transition: border 0.2s ease-in-out;
+    }
+
+    .thumbnail.withlink:hover,
+    .thumbnail.withlink:focus,
+    .thumbnail.withlink.active {
+      border-color: #336699;
+    }
+}
+
+
+
+@media (min-width: 768px) {
+
+    /* show 4 items */
+    .carousel-inner .active,
+    .carousel-inner .active + .carousel-item,
+    .carousel-inner .active + .carousel-item + .carousel-item,
+    .carousel-inner .active + .carousel-item + .carousel-item + .carousel-item {
+        display: block;
+    }
+
+    .carousel-inner .carousel-item.active:not(.carousel-item-right):not(.carousel-item-left),
+    .carousel-inner .carousel-item.active:not(.carousel-item-right):not(.carousel-item-left) + .carousel-item,
+    .carousel-inner .carousel-item.active:not(.carousel-item-right):not(.carousel-item-left) + .carousel-item + .carousel-item,
+    .carousel-inner .carousel-item.active:not(.carousel-item-right):not(.carousel-item-left) + .carousel-item + .carousel-item + .carousel-item {
```

Comment:
mmh not sure what this actually do to be honest. What did you try to achieve here?

Reply:
all this was to get the carousel to display 4 items at a time instead of just one and transition smoothly (e.g. if you remove this section the transition looks weird) ..maybe there are better ways to achieve this, I'm really no expert